site/src/components/mdx/mdx-page-content.tsx

```diff
@@ -143,12 +156,14 @@ export const MdxPageContent: FunctionComponent<MdxPageContentProps> = ({
       }
     };
 
-    window.addEventListener("scroll", onScroll);
+    const throttledOnScroll = throttle(onScroll, 1000);
```

Comment:
We run onScroll to update the url as user scrolls through sections. We don't need to run this as fast as the user scrolls